dowhy/datasets.py

```diff
@@ -52,12 +55,14 @@ def linear_dataset(beta, num_common_causes, num_samples, num_instruments=0,
                    num_treatments = 1,
                    num_frontdoor_variables=0,
                    treatment_is_binary=True,
+                   treatment_is_category=False,
                    outcome_is_binary=False,
                    num_discrete_common_causes=0,
                    num_discrete_instruments=0,
                    num_discrete_effect_modifiers=0,
                    stddev_treatment_noise = 1,
                    one_hot_encode = False):
+    assert not (treatment_is_binary and treatment_is_category)
```

Comment:
this is nice! thanks for the adding the datasets.py example and tests.

Reply:
No problem!